pkg/network/go/bininspect/dwarf.go

```diff
@@ -132,7 +131,11 @@ func (d dwarfInspector) findFunctionDebugInfoEntries(functions []string) (map[st
 	}
 
 	if len(functionsToSearch) != 0 {
-		return nil, errors.New("not all functions found")
+		missingFunctions := make([]string, 0, len(functionsToSearch))
+		for funcName := range functionsToSearch {
+			missingFunctions = append(missingFunctions, funcName)
+		}
+		return nil, fmt.Errorf("not all functions found: %v", missingFunctions)
```

Comment:
along with importing `slices`, `maps`, and `strings`:
```suggestion
		return nil, fmt.Errorf("not all functions found: %s", strings.Join( slices.Collect(maps.Keys(functionsToSearch)), ","))
```